samples/TestApplicationShared/App.cs

```diff
@@ -18,35 +18,35 @@ public class App : Application
     {
         public App()
         {
-			// TODO: I believe this has to happen before we select sub systems. Can we
-			// move this safely into Application itself? Is there anything in here
-			// that is platform specific??
-			//
+            // TODO: I believe this has to happen before we select sub systems. Can we
+            // move this safely into Application itself? Is there anything in here
+            // that is platform specific??
+            //
```

Comment:
I think we can still make it overridable, but the goal was to make general application boilerplate simpler, and for user to not need to worry about all the details. To support this runtime configuration we needed to pull most stuff out of the App constructor. We could clean this up some more, but it's not entirely related to the Skia work.  A good design to reference I think is the new asp.net Startup stuff.